src/i18n/openfire_i18n_en.properties

```diff
@@ -2301,6 +2303,9 @@ ssl.settings.server.label_notrequired=Optional
 ssl.settings.server.label_notrequired_info=Connections between servers may use secured connections.
 ssl.settings.server.label_custom=Custom
 ssl.settings.server.label_custom_info=Advanced configuration
+ssl.settings.server.legacymode.boxtitle=Encrypted (legacy-mode) connections
```

Comment:
We probably want to talk about "Direct TLS" rather than "Legacy-mode".

Reply:
Definitely. We got a an issue in jira to change all that. Until then, I thought to keep things consistent.